ui_testing/pages/order_page.py

```diff
@@ -289,7 +289,7 @@ def create_new_suborder(self, material_type='', article_name='', test_plan='', *
         
         return self.get_suborder_data()
 
-    def duplicate_from_table_view(self, number_of_duplicates=1, index_to_duplicate_from=0):
+    def duplicate_sub_order_table_with_add(self, number_of_duplicates=1, index_to_duplicate_from=0):
```

Comment:
this function is used in two different placed, so I found no need to change its name , and if ther's need to do that , please apply this change in all other usages

Reply:
done